tests/functional/spec/features/repeating_sections/repeating_sections_with_hub_and_spoke.spec.js

```diff
@@ -214,8 +209,6 @@ describe("Feature: Repeating Sections with Hub and Spoke", () => {
       $(ProxyPage.yes()).click();
       $(ProxyPage.submit()).click();
 
-      expect($(DateOfBirthPage.questionText()).getText()).to.equal("What is Samuel Clemens’ date of birth?");
-
```

Comment:
These tests are testing that the question text is proxy or not, so we should keep this expect and remove the rest of the lines of this test. I don't think we need to check two separate pages for the proxy text.

Reply:
As discussed, each section needs to be completed for subsequent tests to follow. We can only therefore remove the interim expects.

Reply:
🤦